compute_tools/src/configurator.rs

```diff
@@ -4,11 +4,13 @@ use std::thread;
 use tracing::{error, info, instrument};
 
 use compute_api::responses::ComputeStatus;
+use utils::id::TenantTimelineId;
 
 use crate::compute::ComputeNode;
 
-#[instrument(skip_all)]
-fn configurator_main_loop(compute: &Arc<ComputeNode>) {
+// Log ttid everywhere
+#[instrument(name = "", fields(ttid = %ttid), skip_all)]
```

Comment:
unsure if we need to set the target as empty..

Now all different background activities will "look to have the same context".

Instead it might be better to create a top level `compute_ctl` span in the `main`, then use that as a parent span for these background spans. This would align with the fact that we do join most of these.

Then only the `compute_ctl` (or whatever we call the top level span) could have the ttid or tenant_id= timeline_id=.

Reply:
Using it as a parent span would practically be:

```patch
+// this will automatically create parent-child relationship to current span
+// which has ttid/tenant_id,timeline_id
+let span = info_span!("configurator");
 std::thread::spawn(move || {
+  let _e = span.entered();
   do_stuff();
 });
```

I think compute_ctl also has async parts, so this isn't as easy. Usage of `entered` in the `async fn main` need to checked as well.